ngraph/python/src/pyngraph/node.cpp

```diff
@@ -34,12 +34,29 @@ using PyRTMap = std::map<std::string, std::shared_ptr<ngraph::Variant>>;
 
 PYBIND11_MAKE_OPAQUE(PyRTMap);
 
+template <typename T>
+void _set_with_variant(PyRTMap& m, const std::string& k, const T v)
+{
+    auto new_v = std::make_shared<ngraph::VariantWrapper<T>>(ngraph::VariantWrapper<T>(v));
+    auto it = m.find(k);
+    if (it != m.end())
+        it->second = new_v;
+    else
+        m.emplace(k, new_v);
+}
+
 void regclass_pyngraph_Node(py::module m)
 {
     auto py_map = py::bind_map<PyRTMap>(m, "PyRTMap");
```

Comment:
Let's move the `PyRTMap` to a separate file.

Reply:
Done.